packages/ui/src/contexts/shortcuts.tsx

```diff
@@ -0,0 +1,51 @@
+import {ComponentChildren, createContext} from 'preact';
+import {useContext, useState} from 'preact/hooks';
+
+export type Shortcut = {
+  key: string;
+  action: string;
+  isGlobal?: boolean;
+};
+
+export type ShortcutModules = 'global' | 'timeline' | 'none';
+type ShortcutsByModule = Record<ShortcutModules, Shortcut[]>;
+type ShortcutsState = {
+  currentModule: ShortcutModules;
+  shortcuts: ShortcutsByModule;
+  setCurrentModule?: (module: ShortcutModules) => void;
+};
+
+const initialshortcuts: ShortcutsByModule = {
+  global: [
+    {key: 'Space', action: 'Toggle playback', isGlobal: true},
```

Comment:
Is `isGlobal` necessary?
Since all global keys by definition are in the `global` module, we could just assume that it will never be set as the current module.